src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  width: 100%;
 }
+
+* {
+  --color-line: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  height: 60px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 10px;
+  list-style: none;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
```

Comment:
it is advisable to indicate this somewhere above, and then just follow it, for example, in the `html` or `body` tag